backend/src/main/java/com/cruru/dashboard/service/facade/DashboardFacade.java

```diff
@@ -36,4 +49,56 @@ private ApplyFormCreateRequest toApplyFormCreateRequest(DashboardCreateRequest r
                 request.endDate()
         );
     }
+
+    public DashboardsOfClubResponse findAllDashboardsByClubId(Long clubId) {
+        List<Long> dashboardIds = getDashboardIdsByClubId(clubId);
+        String clubName = clubService.findById(clubId).getName();
+        List<DashboardPreviewResponse> dashboardResponses = dashboardIds.stream()
+                .map(this::createDashboardPreviewResponse)
+                .sorted(Comparator.comparing(DashboardPreviewResponse::endDate))
+                .toList();
+        return new DashboardsOfClubResponse(clubName, dashboardResponses);
+    }
+
+    private List<Long> getDashboardIdsByClubId(Long clubId) {
+        return dashboardService.findAllByClubId(clubId)
+                .stream()
+                .map(Dashboard::getId)
+                .toList();
+    }
+
+    private DashboardPreviewResponse createDashboardPreviewResponse(Long dashboardId) {
+        ApplyForm applyForm = applyFormService.findByDashboardId(dashboardId);
+        List<Applicant> allApplicants = getAllApplicantsByDashboardId(dashboardId);
+        StatsResponse stats = calculateStats(allApplicants);
+        return new DashboardPreviewResponse(
+                dashboardId,
+                applyForm.getTitle(),
+                stats,
+                applyForm.getUrl(),
+                applyForm.getEndDate()
+        );
+    }
+
+    private List<Applicant> getAllApplicantsByDashboardId(Long dashboardId) {
+        List<Process> processes = processService.findAllByDashboardId(dashboardId);
+        return processes.stream()
+                .flatMap(process -> applicantService.findAllByProcess(process)
+                        .stream())
+                .toList();
+    }
+
+    private StatsResponse calculateStats(List<Applicant> allApplicants) {
+        int totalApplicants = allApplicants.size();
+        int totalFails = (int) allApplicants.stream()
+                .filter(Applicant::isRejected)
+                .count();
+        int totalAccepts = (int) allApplicants.stream()
+                .filter(Applicant::isApproved)
+                .count();
+        int totalPending = (int) allApplicants.stream()
+                .filter(Applicant::isPending)
+                .count();
+        return new StatsResponse(totalAccepts, totalFails, totalPending, totalApplicants);
+    }
```

Comment:
```suggestion
    private StatsResponse calculateStats(List<Applicant> allApplicants) {
        int totalApplicants = allApplicants.size();
        int totalFails = allApplicants.stream()
                .filter(Applicant::isRejected)
                .count();
        int totalAccepts = allApplicants.stream()
                .filter(Applicant::isApproved)
                .count();
        int totalPending = allApplicants.stream()
                .filter(Applicant::isPending)
                .count();
        return new StatsResponse(totalAccepts, totalFails, totalPending, totalApplicants);
    }
```

StatsResponse의 필드를 long으로 바꾸고 형변환을 없애는게 어떤가요?

Reply:
비즈니스 요구사항상 int 범위를 넘어서지 않는 통계가 도출되는 것을 예상하여 int를 적용하였습니다!

형변환을 제거해야하는 이유를 들을 수 있을까요?

Reply:
형변환 가독성이 (저에게는) 안좋아보였습니다. 

Java는 모르겠습니다만, 형변환 로직 자체가 C언어에서는 오버헤드가 큰 것으로 알고 있습니다. `long 범위를 주어져도 문제가 없다면 형변환을 하면서까지 int로 강제해야할 이유가 있나?`라는 의문이 들었습니다.

Reply:
Type Casting은 parent-child class 간의 up-down casting에는 오버헤드 비용이 발생할 수 있지만, 같은 숫자의 primitive type에서는 고려할 정도로 오버헤드가 아주 사소합니다. 사소한 메모리 사용을 고려하자면 오히려 메모리를 잡아먹는 int를 사용하므로써 상쇄될 수 있지 않을까 싶습니다!
우선은 그대로 진행하겠습니다